airbyte-integrations/bases/connector-acceptance-test/unit_tests/data/docs/correct.md

```diff
@@ -84,48 +84,11 @@ The GitHub source connector supports the following [sync modes](https://docs.air
 
 This connector outputs the following full refresh streams:
 
-- [Assignees](https://docs.github.com/en/rest/issues/assignees?apiVersion=2022-11-28#list-assignees)
```

Comment:
Github may have changed their rate limiting policies, but because we include so many github requests to various docs and links, we end up failing the unit_test run because the links are flagged as invalid in `test_documentation.py`. 

I mostly removed a lot of duplicate links across each of the test runs, but these aren't actually related to my fix